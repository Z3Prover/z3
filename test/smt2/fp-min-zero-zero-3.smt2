; Copyright (c) 2015 Microsoft Corporation

(set-logic QF_FP)
(set-info :status sat)

(define-sort FPN () (_ FloatingPoint 11 53))
(declare-const x FPN)
(declare-const y FPN)

(assert (= x (fp.min (_ +zero 11 53) (_ -zero 11 53))))
(assert (= y (fp.min (_ -zero 11 53) (_ +zero 11 53))))
(assert (not (= x y)))

(check-sat)
(get-value (x))
(get-value (y))
(check-sat-using smt)
(get-value (x))
(get-value (y))
(exit)
