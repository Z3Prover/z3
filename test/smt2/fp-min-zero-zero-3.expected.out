sat
((x (_ +zero 11 53)))
((y (_ -zero 11 53)))
sat
((x (_ -zero 11 53)))
((y (_ +zero 11 53)))
